lib/globalStateManagement/themeManagement.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'themes.dart';
+
+class ThemeManagement with ChangeNotifier {
+  ThemeData _currentTheme = mainTheme;
+
+  ThemeData get currentTheme => _currentTheme;
+
+  void toggleTheme() {
+    _currentTheme = _currentTheme == mainTheme ? secondTheme : mainTheme;
```

Comment:
you can use something like this instead:

```
final themes = [primaryTheme, secondaryTheme];
int currentTheme = 0;

Theme get currentTheme => themes[currentTheme];

void toggle() {
  currentTheme = (currentTheme + 1) % themes.length;
}
```

this will allow you to cycle through themes, and adding new themes to `themes` array without changing a code inside your `toggle`